### PR TITLE
Add blazor template, update all to 8.0

### LINKED ISSUE #38507: Add the Blazor Web App (`blazor`) template
### Type of issue

Missing information

### Description

Add the Blazor Web App (`blazor`) template for >=8.0.

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates#blazor

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-new-sdk-templates.md

### Document Version Independent Id

af236ebe-5a79-fb2b-42a0-e5720ee732ae

### Article author

@tdykstra

### Metadata

* ID: 267c314a-e630-097f-edc6-d417be6db191 
* Product: **dotnet-fundamentals**

---
[Associated WorkItem - 187060](https://dev.azure.com/msft-skilling/Content/_workitems/edit/187060)

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #38507 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/dotnet-new-sdk-templates.md](https://github.com/dotnet/docs/blob/100dc4a92d51eddba7c294f41209766e8def7a8e/docs/core/tools/dotnet-new-sdk-templates.md) | [.NET default templates for dotnet new](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates?branch=pr-en-us-38662) |

### CODE REVIEW

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 1]
```diff
@@ -325,6 +331,60 @@ The ability to create a project for an earlier TFM depends on having that versio
...
+  Excludes *launchSettings.json* from the generated template.
+
+- **`-int|--interactivity <None|Server|Webassembly|Auto >`**
+
+  Which interactive render mode to use for interactive components. The possible values are:
+
+  - `None` - No interactivity (static server rendering only)
+  - `Server` - (Default) Runs on the server
+  - `WebAssembly` - Runs in the browser using WebAssembly
+  - `Auto` -Uses Server while downloading WebAssembly assets, then uses WebAssembly
```
Comment (guardrex) [inline docs/core/tools/dotnet-new-sdk-templates.md:357]:
```suggestion
  - `Auto` - Uses interactive server-side rendering while downloading the Blazor bundle and activating the Blazor runtime on the client, then uses client-side rendering with WebAssembly.
```

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 2]
```diff
@@ -325,6 +331,60 @@ The ability to create a project for an earlier TFM depends on having that versio
...
+- **`--exclude-launch-settings`**
+
+  Excludes *launchSettings.json* from the generated template.
+
+- **`-int|--interactivity <None|Server|Webassembly|Auto >`**
+
+  Which interactive render mode to use for interactive components. The possible values are:
+
+  - `None` - No interactivity (static server rendering only)
+  - `Server` - (Default) Runs on the server
```
Comment (guardrex) [inline docs/core/tools/dotnet-new-sdk-templates.md:355]:
```suggestion
  - `Server` - (Default) Runs the app on the server with interactive server-side rendering.
```

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 3]
```diff
@@ -325,6 +331,60 @@ The ability to create a project for an earlier TFM depends on having that versio
...
+  - `WebAssembly` - Runs in the browser using WebAssembly
```
Comment (guardrex) [inline docs/core/tools/dotnet-new-sdk-templates.md:356]:
```suggestion
  - `WebAssembly` - Runs the app using client-side rendering in the browser with WebAssembly.
```

**Review by guardrex — APPROVED** [review]
Just a few nits here and there. Thanks for fixing this up ... I can cross-link the Blazor *Project Structure* topic after these updates go live :tada:.

File: docs/core/tools/dotnet-new-sdk-templates.md [hunk 4]
```diff
@@ -325,6 +331,60 @@ The ability to create a project for an earlier TFM depends on having that versio
...
+
+  Doesn't execute an implicit restore during project creation.
+
+- **`--exclude-launch-settings`**
+
+  Excludes *launchSettings.json* from the generated app.
+
+- **`-int|--interactivity <None|Server|Webassembly|Auto >`**
+
+  Which interactive render mode to use for interactive components. The possible values are:
```
Comment (gewarren) [inline docs/core/tools/dotnet-new-sdk-templates.md:352]:
```suggestion
  Specifies which interactive render mode to use for interactive components. The possible values are:
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**